res/controllers/Denon-MC7000-scripts.js

```diff
@@ -207,8 +207,13 @@ MC7000.init = function() {
         }
     };
 
-    // Activate Softtakeover for Pitch Faders
-    for (i = 1; i <= 4; i++) {
+    // Activate Timer for Softtakeover to avoid conflicts with ControllerStatusSysex
+    engine.beginTimer(2000, "MC7000.delayedSoftTakeover", true);
```

Comment:
```suggestion
    engine.beginTimer(2000, MC7000.delayedSoftTakeover, true);
```
Strings are not functions. Don't use that legacy hack.

Reply:
Thanks a lot for correction. I took this idea from https://github.com/mixxxdj/mixxx/blob/629129a6786bc55b8050881d3405f29e1d94d717/res/controllers/Hercules-DJ-Console-4-Mx-scripts.js#L346